api/src/lib/generateSearchFilters.ts

```diff
@@ -0,0 +1,14 @@
+import { SQL, ilike, or } from "drizzle-orm";
+import { SQLiteColumn } from "drizzle-orm/sqlite-core";
+
+export const generateSearchFilters = (filters: { col: SQLiteColumn; val?: string | null }[]) => {
```

Comment:
You can just update `generateLikeFilters` to this instead of creating a new file. Looks like it'll work for the other applications of it.